### ISSUE TITLE
Replace `-AassumeAssertionsAreEnabled` / `-AassumeAssertionsAreDisabled` with one option

### ISSUE THREAD

**Issue by aosen-xiong** [eisop/checker-framework#2091]
Assertion handling uses two separate flags, and passing both is an error.

```bash
javac -processor org.checkerframework.checker.nullness.NullnessChecker -AassumeAssertionsAreEnabled -AassumeAssertionsAreDisabled Foo.java
```

Output:
```
error: Assertions cannot be assumed to be enabled and disabled at the same time.
```

Expected: a single option, e.g. `-AassumeAssertionsAreEnabled=true|false`, where leaving it out keeps today's default of accounting for both cases. A mode such as `-Amode=jspecify` could then set a default that a command-line value overrides, without special-casing the other flag.

**Comment by wmdietl** [comment]
Wouldn't that single option need three options? We either assume assertions are enabled, they are disabled, or we make neither assumption. Making both assumptions is what we need to forbid.
What happens if neither assumption is enabled?
